app/v4/createOrAmendCodingOut/CreateOrAmendCodingOutController.scala

```diff
@@ -66,7 +62,7 @@ class CreateOrAmendCodingOutController @Inject() (val authService: EnrolmentsAut
         RequestHandler
           .withValidator(validator)
           .withService(service.amend)
-          .withHateoasResult(hateoasFactory)(CreateOrAmendCodingOutHateoasData(nino, taxYear))
+          .withNoContentResult()
```

Comment:
Remove this the request is defaulted to this already [here](https://github.com/hmrc/self-assessment-accounts-api/blob/fafaca58b20d6d8cf81a18706ae7c249a4bea516/app/shared/controllers/RequestHandler.scala#L59).

This method is for when you want to map the result to another status